libs/community/langchain_community/vectorstores/sqlserver.py

```diff
@@ -322,14 +324,118 @@ def distance_strategy(self, value: DistanceStrategy) -> None:
 
     @classmethod
     def from_texts(
-        cls: Type[VST],
+        cls: Type[SQLServer_VectorStore],
         texts: List[str],
         embedding: Embeddings,
         metadatas: Optional[List[dict]] = None,
+        *,
+        connection_string: str,
+        embedding_length: int,
+        table_name: str,
+        db_schema: Optional[str] = None,
+        distance_strategy: DistanceStrategy = DEFAULT_DISTANCE_STRATEGY,
+        ids: Optional[List[str]] = None,
         **kwargs: Any,
-    ) -> VST:
-        """Return VectorStore initialized from texts and embeddings."""
-        return super().from_texts(texts, embedding, metadatas, **kwargs)
+    ) -> SQLServer_VectorStore:
+        """Create a SQL Server vectorStore initialized from texts and embeddings.
+        Args:
+            connection_string: SQLServer connection string.
```

Comment:
The order of description of the args doesnt match the order specified in the function definition.